created function showSalary

AleksandrMenshchikov opened the pull request.

AleksandrMenshchikov committed:
created function showSalary

AleksandrMenshchikov committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20200127-2_aleksandr-menshchikov1982

AleksandrMenshchikov committed:
created function getMinMax

jsru commented:
Не удалось выполнить проверку решения, пожалуйста, убедитесь в том, что все тесты
          выполняются локально, а также проверьте, нет ли в решении отладочных вызовов console.log
          (в результате вызова тестов в консоли не должно быть никакого лишнего вывода).

AleksandrMenshchikov committed:
update

AleksandrMenshchikov committed:
update

AleksandrMenshchikov committed:
updated
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20200127-2_aleksandr-menshchikov1982

AleksandrMenshchikov committed:
update

AleksandrMenshchikov committed:
update

AleksandrMenshchikov committed:
deleted

jsru commented:
Пожалуйста, не меняйте «системные» файлы.
          Задача не может считаться решенной, если были изменены файлы тестов или другие «системные» файлы.
          Следующие системные файлы были изменены:
```diff
- 3-module/2-task/README.md — удален
- 3-module/2-task/task.test.js — удален
```
